lib/flags.js

```diff
@@ -1,27 +1,29 @@
 const axios = require("axios");
 require("dotenv").config();
 
-let flags = [];
+class FlagManager {
+  constructor() {
+    this._flags = [];
```

Comment:
Any particular reason for using an `_` here? No problems with it, just wondering

Reply:
Yeah, I noticed it in Pioneer and have also seen it around in other places. It's a semantic thing that denotes that it's sort of private. Like whoever uses the api shouldn't be messing with variables with `_` directly, and they should be using the methods to interact with the variable.

From an article: "The use of underscores to designate secret methods is a typical trend in languages without access modifiers"